python/app/qt_importer.py

```diff
@@ -0,0 +1,37 @@
+# NOTE: This repo is typically used as a Toolkit app, but it is also possible use the console in a
```

Comment:
Missing copyright!

Reply:
Good catch, I'm not used to writing new modules yet :)